### PR TITLE
Add in note on QAT memory consumption for sparse transer learning

### PR DESCRIPTION
**Pull request by markurtz**

### CODE REVIEW

File: integrations/ultralytics-yolov3/tutorials/yolov3_sparse_transfer_learning.md [hunk 1]
```diff
@@ -137,6 +137,10 @@ The recipes are specific to the sparsification type, so the training command wil
         ```bash
         python train.py --data voc.yaml --cfg ../models/yolov3-spp.lrelu.yaml --weights DOWNLOADED_PATH --hyp data/hyp.finetune.yaml --recipe ../recipes/yolov3-spp.transfer_learn_pruned_quantized.md
         ```
+     Note, running quantization aware training (QAT) will consume up to three times the memory as FP32 training. 
+     It is recommended to use this recipe with a batch size that fits in memory for FP32 as normal at the start, though.
```
Comment (bfineran) [inline integrations/ultralytics-yolov3/tutorials/yolov3_sparse_transfer_learning.md:141]: why not FP16 if the script will use that by default?

File: integrations/ultralytics-yolov3/tutorials/yolov3_sparse_transfer_learning.md [hunk 2]
```diff
@@ -137,6 +137,10 @@ The recipes are specific to the sparsification type, so the training command wil
         ```bash
         python train.py --data voc.yaml --cfg ../models/yolov3-spp.lrelu.yaml --weights DOWNLOADED_PATH --hyp data/hyp.finetune.yaml --recipe ../recipes/yolov3-spp.transfer_learn_pruned_quantized.md
         ```
+     Note, running quantization aware training (QAT) will consume up to three times the memory as FP32 training.
```
Comment (jeanniefinks) [inline integrations/ultralytics-yolov3/tutorials/yolov3_sparse_transfer_learning.md:140]:
```suggestion
     Note: Running quantization-aware training (QAT) will consume up to three times the memory as FP32 training. 
```

**Review by jeanniefinks — DISMISSED** [review]
Approving but w/ with the three minor edits you can make!

File: integrations/ultralytics-yolov3/tutorials/yolov3_sparse_transfer_learning.md (hunk 1 above)
Comment (markurtz) [reply integrations/ultralytics-yolov3/tutorials/yolov3_sparse_transfer_learning.md]: yep, good catch, that should say fp16

**Review by bfineran — APPROVED** [review]

File: integrations/ultralytics-yolov3/tutorials/yolov3_sparse_transfer_learning.md [hunk 3]
```diff
@@ -137,6 +137,10 @@ The recipes are specific to the sparsification type, so the training command wil
         ```bash
         python train.py --data voc.yaml --cfg ../models/yolov3-spp.lrelu.yaml --weights DOWNLOADED_PATH --hyp data/hyp.finetune.yaml --recipe ../recipes/yolov3-spp.transfer_learn_pruned_quantized.md
         ```
+     Note: Running quantization-aware training (QAT) will consume up to three times the memory as FP32 training. 
+     It is recommended to use this recipe with a batch size that fits in memory for FP32 as normal at the start, though.
```
Comment (bfineran) [inline integrations/ultralytics-yolov3/tutorials/yolov3_sparse_transfer_learning.md:141]:
```suggestion
     Note: Running quantization-aware training (QAT) will consume up to three times the memory as FP16 training. 
     It is recommended to use this recipe with a batch size that fits in memory for FP16 as normal at the start, though.
```

**Review by bfineran — CHANGES_REQUESTED** [review]
sorry, saw comment only

**Pull request merged**